### PR TITLE
Add mongodb Cursor and CursorInterface stubs and fix WriteConcern::__construct

### PR DESCRIPTION
**Pull request by franmomu**
Stubs for:

https://www.php.net/manual/en/class.mongodb-driver-cursorinterface.php
https://www.php.net/manual/en/class.mongodb-driver-cursor.php

**UPDATE**: I've also updated `WriteConcern::__construct` based on https://www.php.net/manual/es/mongodb-driver-writeconcern.construct.php

### CODE REVIEW

File: stubs/mongodb.phpstub [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     * @return array<TValue>
+     */
+    public function toArray();
+}
+
+/**
+ * @template-covariant TValue of array|object
+ *
+ * @template-implements Iterator<int, TValue>
+ * @template-implements CursorInterface<int, TValue>
```
Comment (franmomu) [inline stubs/mongodb.phpstub:23]: The types are based on 
`array|object`: https://www.php.net/manual/en/mongodb-driver-cursor.current.php
`int`: https://www.php.net/manual/en/mongodb-driver-cursor.key.php

**Comment by muglug** [comment]
You need to add logic in `Config.php` that'll load the stubs if the mongodb extension is enabled

**Comment by franmomu** [comment]
> You need to add logic in `Config.php` that'll load the stubs if the mongodb extension is enabled

Makes sense, thanks!

**Comment by muglug** [comment]
Thanks!

**Pull request merged**